src/cartridges/int_adyen_SFRA/cartridge/templates/default/checkout/billing/adyenComponentForm.isml

```diff
@@ -2,65 +2,108 @@
-        originKey: '${pdict.adyen.originKey}',
-        environment: '${pdict.adyen.environment}'
-    };
-    window.installments = '${pdict.adyen.installments}';
-    window.paypalMerchantID = '${pdict.adyen.paypalMerchantID}';
-    window.googleMerchantID = '${pdict.adyen.googleMerchantID}';
-    window.cardholderNameBool = '${pdict.adyen.cardholderNameBool}';
-    window.paypalIntent = '${pdict.adyen.paypalIntent}';
-    window.merchantAccount = '${pdict.adyen.merchantAccount}';
-    var showStoreDetails = ${customer.authenticated && adyenRecurringPaymentsEnabled};
 
-    window.getPaymentMethodsURL = "${URLUtils.https('Adyen-GetPaymentMethods')}";
-    window.paymentFromComponentURL = "${URLUtils.https('Adyen-PaymentFromComponent')}";
+<script>
+      window.Configuration = {
+          locale: '${request.locale}',
+          originKey: '${pdict.adyen.originKey}',
+          environment: '${pdict.adyen.environment}',
+      };
+      window.adyenClientKey = '${pdict.adyen.adyenClientKey}';
+      window.amazonMerchantID = '${pdict.adyen.amazonMerchantID}';
+      window.amazonStoreID = '${pdict.adyen.amazonStoreID}';
+      window.amazonPublicKeyID = '${pdict.adyen.amazonPublicKeyID}';
 </script>
 
-<isset name="AdyenHelper" value="${require('*/cartridge/scripts/util/adyenHelper')}" scope="pdict"/>
-<isset name="ratePayValues" value="${pdict.AdyenHelper.getRatePayID()}" scope="page"/>
+<isif condition="${request.getHttpQueryString().indexOf('amazonCheckoutSessionId') > -1} ">
+    <link rel="stylesheet" type="text/css" href="${pdict.AdyenHelper.getCheckoutCSS()}"/>
+    <isscript>
+        var assets = require('*/cartridge/scripts/assets.js');
+        assets.addJs('/js/amazon.js');
+    </isscript>
 
-<isset name="ShowConfirmationPaymentFromComponent" value="${URLUtils.https('Adyen-ShowConfirmationPaymentFromComponent')}" scope="page"/>
+    <script>
+        var queryString = '${request.getHttpQueryString()}'.split("=");
+        var amazonCheckoutSessionId = queryString[queryString.length - 1];
+        window.paymentFromComponentURL = "${URLUtils.https('Adyen-PaymentFromComponent')}";
+        window.amazonCheckoutSessionId = amazonCheckoutSessionId;
+        window.returnURL = "${URLUtils.https('Checkout-Begin', 'stage', 'payment')}";
+    </script>
+    <div id="amazon-container"></div>
+     <input id="adyenStateData" type="hidden" name="${adyenPaymentFields.adyenStateData.htmlName}"/>
 
+     <div class="modal fade" id="action-modal" role="dialog">
+         <div class="modal-dialog" id="adyenModalDialog">
+             <div id="action-container"></div>
+         </div>
+     </div>
 
-<input id="adyenStateData" type="hidden" name="${adyenPaymentFields.adyenStateData.htmlName}"/>
-<input type="text" class="form-control cardNumber" style="display:none" id="cardNumber" name="${creditFields.cardNumber.htmlName}">
-<input type="hidden" class="form-control" id="cardType" name="${creditFields.cardType.htmlName}">
+      </form>
```

Comment:
There does not seem to be a corresponding form opening tag

Reply:
Feels like this must be an opening tag indeed rather than closing